src/d3d9/d3d9_util.cpp

```diff
@@ -55,11 +55,8 @@ namespace dxvk {
     const auto& limits = pDevice->properties().core.properties.limits;
     VkSampleCountFlags supportedSampleCounts = limits.framebufferColorSampleCounts & limits.framebufferDepthSampleCounts;
 
-    while (sampleCount > supportedSampleCounts)
-      sampleCount >>= 1;
-
-    if (pSampleCount)
-      *pSampleCount = VkSampleCountFlagBits(sampleCount);
+    if (sampleCount > supportedSampleCounts)
```

Comment:
should probably check `&` here since it's legal for an implementation to e.g. support 4x but not 2x.